routes/web.php

```diff
@@ -1,7 +1,11 @@
 <?php
 
 use Illuminate\Support\Facades\Route;
+use App\Http\Controllers\CommentController;
```

Comment:
Remember to use the comment in the `App/Http/Controllers/Api` namespace after the review above.